### PR TITLE
#166239483 Fix gulp error in travis.yml

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
#### What does this PR do?
Have Tests running successfully on Travis CI

#### Description of Task to be completed?
This Pull request is about setting up the Workout Manager (Wger) project to successfully run tests on Travis CI. The intent was to fix the project Python versioning and Gulp declaration in the .travis file.

#### What are the relevant pivotal tracker stories?
[#166239483](https://www.pivotaltracker.com/story/show/166239483)

#### Screenshots (if appropriate)
Screenshot 2019-05-28 at 09 58 35

### CODE REVIEW

**Comment by Emmanuel-Dominic** [comment]
@Manorlds-Eaglespark , kindly review your commit message for better workflow, `commit type`,`commit body`,`commit subject` ,`commit footer`.
Screenshot 2019-05-28 at 10 12 43

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
 #
 # Requirements for wger for production
 #
-
-Django>=1.9,<1.10
-django-recaptcha
-reportlab
+Django==1.9   # changed this line
+django-recaptcha==1.3.1  # changed this line
```
Comment (Rhytah) [inline requirements.txt]: You can remove these comments in the file for now according to another task, we are going to have explicitly defined requirements versions to install.

File: .travis.yml [hunk 2]
```diff
@@ -43,11 +41,12 @@ install:
...
 
   # Setup application
-  - if [[ "$DB" = "sqlite" ]]; then invoke create_settings; fi
-  - if [[ "$DB" = "postgresql" ]]; then invoke create_settings --database-type postgresql; fi
+  - if [[ "$DB" = "sqlite" ]]; then invoke create-settings; fi
+  - if [[ "$DB" = "postgresql" ]]; then invoke create-settings --database-type postgresql; fi
 
 # Create test databases
 before_script:
+  - npm install --global gulp-cli
```
Comment (Rhytah) [inline .travis.yml]: Great work on fixing the gulp linting, the build is working as it should successfully.

Comment (Manorlds-Eaglespark) [reply .travis.yml]: Thanks @Rhytah

**Review by reifred — COMMENTED** [review]
Good work @Manorlds-Eaglespark for fixing the gulp errors, the build is now running, however, look into changes requested by @Rhytah for removing the unnecessary comments.

**Review by Emmanuel-Dominic — APPROVED** [review]
@Manorlds-Eaglespark, Task successfully is done well now, gulp errors are fixed and the build is now running.
However, Note requested changes by @Rhytah for removing the unnecessary comments.

**Review by Rhytah — APPROVED** [review]

**Review by reifred — APPROVED** [review]

File: wger/utils/language.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def load_language(language_code=None):
         language = Language.objects.get(short_name=used_language)
     except ObjectDoesNotExist:
         # No luck, load english as our fall-back language
-        language = Language.objects.get(short_name="en")
+        language, is_created = Language.objects.get_or_create(short_name=used_language)
```
[the remark this answers is not in the document]
Comment (Manorlds-Eaglespark) [reply wger/utils/language.py]: No, it was not, but it has been resolved. Thanks @AGMETEOR

[the remark this answers is not in the document]
Comment (AGMETEOR) [reply wger/utils/language.py]: @Manorlds-Eaglespark I am trying to understand why `language, is_created = Language.objects.get_or_create(short_name=used_language)` is a relevant change to this story?

[the remark this answers is not in the document]
Comment (Manorlds-Eaglespark) [reply wger/utils/language.py]: Oh yes, thanks it has been fixed.

**Pull request merged**